### PR TITLE
Fix contribute button style in podcast episode page

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Before
screen shot 2018-11-26 at 11 37 30

After
screen shot 2018-11-26 at 11 37 18

### CODE REVIEW

File: static/src/stylesheets/module/journalism/_media.scss [hunk 1]
```diff
@@ -302,9 +302,19 @@
     border-radius: 30px;
     height: 34px;
 
-    .from-content-api.from-content-api {
+    .from-content-api.from-content-api & {
```
Comment (ajwl) [inline static/src/stylesheets/module/journalism/_media.scss]: I haven't seen this convention to add `&` at the end of a parent selector. Normally I see it at the start of a child selector (is it just a different way of writing this?

File: static/src/stylesheets/module/journalism/_media.scss [hunk 2]
```diff
@@ -302,9 +302,19 @@
...
+        background: $highlight-main;
+        color: $brightness-7;
+
+        &:hover,
+        &:focus {
+            background: darken($highlight-main, 5);
+        }
+
+        .inline-icon {
+            fill: currentColor;
```
Comment (ajwl) [inline static/src/stylesheets/module/journalism/_media.scss]: And sorry I guess I missed this somewhere too, but where is `currentColor` defined?

Comment (regiskuckaertz) [reply static/src/stylesheets/module/journalism/_media.scss]: It's a CSS keyword [meaning](https://css-tricks.com/currentcolor/) the current `color` by looking at the element or its ancestors.

**Comment by ajwl** [comment]
👍 all works

File: static/src/stylesheets/module/journalism/_media.scss (hunk 2 above)
Comment (ajwl) [reply static/src/stylesheets/module/journalism/_media.scss]: ooh good to know. thanks

**Pull request merged**